{"name": "test_broken", "status": "broken", "statusDetails": {"message": "Exception: oops", "trace": "def test_broken():\n>       raise Exception('oops')\nE       Exception: oops\n\nallure.py:20: Exception"}, "start": 1612252156381, "stop": 1612252156382, "uuid": "b30e068d-518b-46bd-9384-c853d1532561", "historyId": "d2b935e4eb4fb71605a04af7c4aa8aec", "testCaseId": "b681323366c3c26c23e8a122b8f3080f", "fullName": "allure#test_broken", "labels": [{"name": "suite", "value": "allure"}, {"name": "host", "value": "SC-202009151434"}, {"name": "thread", "value": "53232-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "allure"}]}